client/js/components/button/BackToTopButton.vue

```diff
@@ -52,7 +53,7 @@ export default {
       this.scrollPos = document.documentElement.scrollTop
       const fromBottom = (this.contentHeight - 10 - this.footerheight > this.scrollPos) ? 10 : -this.contentHeight + 10 + this.footerheight + this.scrollPos
 
-      this.buttonPosition = `bottom: ${fromBottom}px; left: ${this.positionFromLeft}px; position: fixed`
+      this.buttonPosition = `bottom: ${fromBottom}px; left: ${this.positionFromLeft -80}px`
```

Comment:
I can't test it, but I think 80 is a bit much. And maybe we have to change the `containerElement` too, not only the ``observer. DId You check that?